apps/src/netsim/NetSimVisualization.js

```diff
@@ -192,6 +198,7 @@ NetSimVisualization.prototype.setLocalNode = function (newLocalNode) {
     this.localNode.isLocalNode = true;
   } else {
     this.localNode.kill();
+    this.localNode = null;
```

Comment:
This fixes a visualization bug that happened when a client was kicked - once you've called kill() on a viznode it's effectively invalidated and shouldn't be interacted with anymore (even though it takes a second to animate out), but we were hanging onto a reference to it and trying to update it when the client reconnected.  In fact, we want to drop our reference to this node so we make a whole new one on reconnect.